mappings/net/minecraft/client/font/PlayerHeadGlyphs.mapping

```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_11879 net/minecraft/client/font/PlayerHeadGlyphs
 	CLASS 1
 		METHOD load (Ljava/lang/Object;)Ljava/lang/Object;
 			ARG 1 source
-	CLASS class_11880 Glyph
+	CLASS class_11880 HeadGlyph
```

Comment:
Inner class names shouldn't conflict with classes in the same package (though I doubt it messes up decomp output)